build/pxc-configure-pxc.sh

```diff
@@ -69,6 +69,10 @@ MYSQL_VERSION=$(mysqld -V | awk '{print $3}' | awk -F'.' '{print $1"."$2}')
 if [ "$MYSQL_VERSION" == '8.0' ]; then
 	grep -E -q "^[#]?admin-address" "$CFG" || sed '/^\[mysqld\]/a admin-address=\n' ${CFG} 1<>${CFG}
 	grep -E -q "^[#]?log_error_suppression_list" "$CFG" || sed '/^\[mysqld\]/a log_error_suppression_list="MY-010055"\n' ${CFG} 1<>${CFG}
+    if [ "$HIGH_COMPRESSION" == 'yes' ]; then
+        sed -r "s|^[#]?default_storage_engine=.*$|default_storage_engine=BLACKHOLE|" ${CFG} 1<>${CFG}
```

Comment:
We have the same in PBM https://github.com/percona/percona-backup-mongodb/tree/main/pbm/storage/blackhole
Exciting and effective stuff, but careful using it. We've lost a few clusters over the event horizon.

Reply:
And I don't fully understand which type of black hole is used?
Schwarzschild black hole would grow in mass (hence maintenance and retention issues).
And with Kerr black holes there is a possibility of some time-related issues (see. https://arxiv.org/pdf/0706.0622v3.pdf) 
I suggest proper testing.